src/content/en/tools/workbox/guides/configure-workbox.md

```diff
@@ -2,10 +2,11 @@ project_path: /web/tools/workbox/_project.yaml
 book_path: /web/tools/workbox/_book.yaml
 description: A guide on how to configure Workbox.
 
-{# wf_updated_on: 2017-12-01 #}
+{# wf_updated_on: 2017-12-13 #}
 {# wf_published_on: 2017-11-15 #}
+{# wf_blink_components: Blink>ServiceWorker #}
 
-# Configure Workbox {: .page-title }
+# Configure the Runtime Options {: .page-title }
```

Comment:
I'd like to avoid this phrasing. I don't know why but I think it conflicts with Runtime caching in my head and doesn't fit in with the current flow of the guides.